provisioners/file/home/ec2-user/bin/limit-logs

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+cd /data/jenkins/jobs
+sudo find . -type f -name '*log' -size +15M \
```

Comment:
You don't need the `cd` if you just `find /data/jenkins/jobs -type …`
Also, does that actually need sudo? Shouldn't the logs be owned by jenkins?

Reply:
Actually I ran into permission errors without the sudo. Lets see if I still have them in the terminal buffer.

Reply:
```
ec2-user@ip-172-31-37-206:/data/jenkins/jobs> limit-logs 
truncate: cannot open './scf-master-sles/builds/56/110.log' for writing: Permission denied
truncate: cannot open './scf-master-sles/builds/56/log' for writing: Permission denied
truncate: cannot open './scf-develop/builds/267/log' for writing: Permission denied
truncate: cannot open './scf-develop/builds/267/95.log' for writing: Permission denied
truncate: cannot open './scf/branches/PR-1436/builds/1/89.log' for writing: Permission denied
truncate: cannot open './scf/branches/PR-1436/builds/1/log' for writing: Permission denied
```

Actually these might have been the logs you truncated before, they were all of size 101M.
And if you did that via root or sudo, then they may have become root owned.

Checking ownership of the logs now ...

Reply:
Ah! Logs are owned by jenkins. I am however user `ec2-user`.
Lets see if I can shift the cron job over to user `jenkins`.

Reply:
Changed with commit 5f4ac1ddbc5349bd9d607afb808001070b44de43